Readme.rst

```diff
@@ -49,11 +49,11 @@ Contributing
 
 If you want to contribute to Neos and want to set up a development environment, then follow these steps:
 
-``composer create-project neos/neos-development-distribution neos-development dev-master --keep-vcs``
+``composer create-project neos/neos-development-distribution neos-development dev-master --keep-vcs --prefer-install=auto``
```

Comment:
the --keep-vcs is no longer needed then right?
Or is this for legacy composer?

Reply:
I *think* it is still needed (https://getcomposer.org/doc/03-cli.md#create-project) though I'm unsure how composer 2.x really behaves for source install. At least it's not stated in the CLI option that it directly correlates with prefer-install